### ISSUE TITLE
Accessibility validation for Caption in table fail

### ISSUE THREAD

**Issue by fenix4** [cksource/ckeditor-plugin-a11ychecker#243]
I have this code in a CKEditor, as you see it doesn't has any caption attribute: 

```
<table>
	<tbody>
		<tr>
			<th>Stade</th>
			<th>Programs</th>
		</tr>
		<tr>
			<td><strong>DF</strong></td>
			<td>
			<p>286</p>
			</td>
		</tr>
		<tr>
			<td>TOTAL</td>
			<td>
			<p>2405</p>
			</td>
		</tr>
	</tbody>
</table>
```
In my /libraries/a11ychecker/libs/quail/tests.json, and it have testability = 1

```
  "tableCaptionIdentifiesTable": {
    "type": "selector",
    "testability": 1,
    "title": {
      "de": "Überschriften sollten ihre Tabelle identifizieren",
      "en": "Captions should identify their table",
      "nl": "Beschrijvingen moeten hun tabellen identificeren"
    },
    "description": {
      "de": "Stellen Sie sicher, dass eine Tabellenüberschrift die Tabelle mit Name, Figurennummer etc. identifiziert",
      "en": "Check to make sure that a table's caption identifies the table with a name, figure number, etc.",
      "nl": "Controleer of de beschrijving van een tabel de tabel identificeert met een naam, nummer en dergelijke."
    },
    "guidelines": {
      "wcag": {
        "1.3.1": {
          "techniques": [
            "H39"
          ]
        }
      }
    },
```
But, I get this message when I press Check Accessibility button :

> The document does not contain any accessibility issues.

Any suggestion, thanks!!
